cas/service.c

```diff
@@ -624,17 +638,28 @@ static int cas_service_sdev_id_set(struct cas_service *cas_svc)
 	return M0_RC(rc);
 }
 
+M0_INTERNAL struct m0_dtm0_domain *
+m0_cas__ut_svc_dtm0_domain_get(struct m0_reqh_service *svc)
+{
+	struct cas_service *service = M0_AMB(service, svc, c_service);
+	return service->c_dtm0_domain;
+}
+
 static int cas_service_start(struct m0_reqh_service *svc)
 {
-	int                  rc;
-	struct cas_service  *service = M0_AMB(service, svc, c_service);
-	struct m0_be_domain *ut_dom;
+	int                    rc;
+	struct cas_service    *service = M0_AMB(service, svc, c_service);
+	struct m0_be_domain   *ut_be_dom;
+	struct m0_dtm0_domain *dod;
 
 	M0_PRE(m0_reqh_service_state_get(svc) == M0_RST_STARTING);
-	ut_dom = m0_cas__ut_svc_be_get(svc);
+	ut_be_dom = m0_cas__ut_svc_be_get(svc);
```

Comment:
I think we need to rename or remove all the "__ut_" functions.
It seems like these code are only used in UT. But they are now used here.

Reply:
I think this can be changed in a separate task.